watertap/core/zero_order_costing.py

```diff
@@ -967,6 +1110,196 @@ def cost_gac(blk):
             blk.unit_model.activated_carbon_demand[t0], "activated_carbon"
         )
 
+    def cost_hydrothermal_gasification(blk):
+        """
+        General method for costing hydrothermal gasification unit. Capital cost
+        is based on the CHG reactor and other wastewater treatment equipments.
+        """
+        t0 = blk.flowsheet().time.first()
+
+        # Get parameter dict from database
+        parameter_dict = blk.unit_model.config.database.get_unit_operation_parameters(
+            blk.unit_model._tech_type, subtype=blk.unit_model.config.process_subtype
+        )
+
+        # Get costing parameter sub-block for this technology
+        (
+            IF_reactor,
+            EP_reactor,
+            F0_reactor,
+            SE_reactor,
+            IF_pump,
+            EP_pump,
+            F0_pump,
+            SE_pump,
+            IF_booster,
+            EP_booster,
+            F0_booster,
+            SE_booster,
```

Comment:
Why are there so many pumps for only one unit?

Reply:
Ah, these must be the different cost parameters for one pump and one booster actually, correct?

Reply:
Yeah, exactly. This unit includes the whole CHG process with reactor, feed pump, booster pump, feed/product exchanger, fired heater, hydrocyclone and air fin cooler.

Reply:
> This unit includes the whole CHG process with reactor, feed pump, booster pump, feed/product exchanger, fired heater, hydrocyclone and air fin cooler.

I think that quote of yours would be perfect to put in the code comments.

Reply:
Yeah, I added this to the comment.